### PR TITLE
IDEMPIERE-6348 Change oomph to 13

### PR DESCRIPTION
**Pull request by CarlosRuiz-globalqss**

### CODE REVIEW

**Comment by CarlosRuiz-globalqss** [comment]
Hi @hengsin - can you please take a look to this?  I'm not sure if that's the correct way to change the oomph file

File: utils_dev/oomph/idempiere.setup [hunk 1]
```diff
@@ -26,7 +26,7 @@
   <setupTask
       xsi:type="setup:VariableTask"
       name="idempiere.target.platform"
-      defaultValue="idempiere-12-local-mirror"
+      defaultValue="idempiere-13-local-mirror"
```
Comment (hengsin) [inline utils_dev/oomph/idempiere.setup:29]: defaultValue must equals to the target name in https://github.com/idempiere/idempiere/blob/master/org.idempiere.p2.targetplatform/org.idempiere.p2.targetplatform.mirror.target

**Review by hengsin — CHANGES_REQUESTED** [review]

File: utils_dev/oomph/idempiere.setup (hunk 1 above)
Comment (CarlosRuiz-globalqss) [reply utils_dev/oomph/idempiere.setup]: done

**Review by hengsin — APPROVED** [review]
looks good

**Pull request merged**